pkg/sql/exec_util.go

```diff
@@ -337,6 +337,13 @@ var experimentalUseNewSchemaChanger = settings.RegisterEnumSetting(
 	},
 )
 
+var experimentalStreamReplicationEnabled = settings.RegisterBoolSetting(
+	"sql.defaults.experimental_stream_replication.enabled",
+	"default value for experimental_stream_replication session setting;"+
+		"disables the ability to setup a replication stream",
```

Comment:
nit: The difference between "enabled" in the setting name, and "disables" in the test confused me a bit. I think "enables the ability..." would be clearer?